preprocessing_qc/process_cfdna.ipynb

```diff
@@ -0,0 +1,876 @@
+{
```

Comment:
<p>you could add more description of what the notebook does and what the data is </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/buddi_atac/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>added in <a href="https://github.com/greenelab/buddi_atac/pull/1/commits/f75d90dc63400fc3a0e34ee2ffb8df26a5a35608" target="_blank">f75d90d</a></p>